Sources/Kitura/Kitura.swift

```diff
@@ -149,6 +164,40 @@ public class Kitura {
         }
         serverLock.unlock()
     }
+    
+    /// Start all registered servers and return the number of servers that failed to start.
+    ///
+    ///### Usage Example: ###
+    /// Make all registered servers start listening on their port.
+    ///```swift
+    /// let router = Router()
+    /// Kitura.addHTTPServer(onPort: 8080, with: router)
+    /// Kitura.startWithStatus() // Returns the number of failed server starts.
+    ///```
+    public class func startWithStatus() -> Int {
```

Comment:
Rather than duplicating code, can we make the existing `start()` call this one, eg:
```swift
public class func start() {
    _ = startWithStatus()
}
```